improver_tests/acceptance/test_nbhood_land_and_sea.py

```diff
@@ -48,7 +48,37 @@ def test_basic(tmp_path):
     input_path = kgo_dir / "input.nc"
     mask_path = kgo_dir / "ukvx_landmask.nc"
     output_path = tmp_path / "output.nc"
-    args = [input_path, mask_path, "--radii", "20000", "--output", output_path]
+    args = [
+        input_path,
+        mask_path,
+        "--neighbourhood-shape",
+        "square",
+        "--radii",
+        "20000",
+        "--output",
+        output_path,
+    ]
+    run_cli(args)
+    acc.compare(output_path, kgo_path)
+
+
+def test_basic_circular(tmp_path):
```

Comment:
Same again, if not writing it in the doc-string, the test above needs a `_square` suffix.